tdi_python/tdiTable.py

```diff
@@ -1165,6 +1169,18 @@ def _get_cont_data_fields(self, info, data_handle):
             content.append(self._process_data_fields(info._cont_data_fields, data_h, True))
         return content
 
+
+    '''
+        Certain fields might need target specific handling.
+        Targets can override this method to handle such cases.
+        Targets should return True/None if a data field is handled in that code so that they can be skipped here.
+    '''
+    def stringify_output_target_specific(self, data_field, value):
```

Comment:
nit: prepend _ to the name since this function is also internal to this class

Reply:
Done